python/friesian/example/ncf/ncf_train.py

```diff
@@ -104,8 +105,8 @@ def build_model(num_users, num_items, class_num, layers=[20, 10], include_mf=Tru
                                init_ray_on_spark=True)
     elif args.cluster_mode == "spark-submit":
         sc = init_orca_context("spark-submit")
-
-    movielens_data = movielens.get_id_ratings("/tmp/movielens/")
+    
+    movielens_data = read_data_sets(args.data_dir)[:,0:3]
```

Comment:
why need to change the API?

Reply:
sorry, I forgot it can work even the data is downloaded